recipes/libassuan/meta.yaml

```diff
@@ -19,7 +19,8 @@ build:
     - make install
   skip: true  # [win]
   run_exports:
-    - libassuan
+    - {{ pin_subpackage('libassuan', max_pin='x.x') }}
```

Comment:
[Looks like](https://abi-laboratory.pro/tracker/timeline/libassuan/) just major should probably be enough.